food-search-app/src/main/java/com/example/foodsearchapp/model/Status.java

```diff
@@ -0,0 +1,26 @@
+package com.example.foodsearchapp.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@Table(name = "status")
+public class Status {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Enumerated(EnumType.STRING)
+    @Column(length = 50)
+    private EStatus name;
+
+    public Status(EStatus name) {
+        this.name = name;
+    }
+
+    public Status() {
```

Comment:
First - I don't see business purpose of this entity now. 
Do not add any code while it's not necessary.

Reply:
Second - I don't see reason to make entity, if I can just add enums set to user